.github/workflows/gitlab-ci.yml

```diff
@@ -0,0 +1,19 @@
+name: Mirror and run GitLab CI
+
+on: [push]
```

Comment:
Is this triggered on any push to any branch, including master?

Reply:
Yes, on a push to any repository branch. We need to switch to `pull_request` to activate it for pull requests.